src/it/resources/org/checkstyle/suppressionxpathfilter/throwscount/SuppressionXpathRegressionThrowsCount3.java

```diff
@@ -0,0 +1,22 @@
+package org.checkstyle.suppressionxpathfilter.throwscount;
+
+public class SuppressionXpathRegressionThrowsCount3 {
+
+    public void myFunc() throws ArithmeticException,
+            NumberFormatException { // ok
```

Comment:
This is again the same as first.
Please make violation on method that is defined in anonymous class.